ratatoskr/app/calendarutil.py

```diff
@@ -68,8 +76,8 @@ def create_calendar_for_schedule(schedule) -> tuple[dict, str]:
         },
         "conferenceData": {
             "createRequest": {
-                "requestId": str(uuid.uuid4()),
-                "conferenceSolutionKey": {"type": "hangoutsMeet"},
+                "requestId": str(uuid.uuid4())
+                # "conferenceSolutionKey": {"type": "hangoutsMeet"},
```

Comment:
Why was this commented out? Somehow this doesn't affect the creation of conferences when I tested the app, my best guess is that this is the default setting, but I don't really understand why this was commented out.

Reply:
Student accounts can't create Google Meets, so this causes an error, as "conferenceSolutionKey" type "hangoutsMeet" doesn't exist for those accounts. I couldn't create any schedules while this was uncommented, and I just forgot to de-comment it back out.